src/main/java/net/atos/zac/app/planitems/PlanItemsRESTService.java

```diff
@@ -211,7 +211,10 @@ public void doHumanTaskplanItem(final RESTHumanTaskData humanTaskData) {
                     Bronnen.fromZaak(zaak)));
         }
         cmmnService.startHumanTaskPlanItem(humanTaskData.planItemInstanceId, humanTaskData.groep.id,
-                                           humanTaskData.medewerker != null ? humanTaskData.medewerker.id : null,
+                                           humanTaskData.medewerker != null && !humanTaskData.medewerker.toString().isEmpty()
```

Comment:
As discussed probably nicer to add validation here which returns a 400 in case the medewerker.id or medewerker.name are empty. The frontend should also be fixed to never sent an empty medewerker I guess.